eng/TestFramework.targets

```diff
@@ -1,12 +1,21 @@
 <Project>
+  <PropertyGroup>
+    <SessionRecordTargetFramework>netcoreapp2.1</SessionRecordTargetFramework>
```

Comment:
Nit: Why not plural, i.e. "SessionRecords..." like all the other properties below?